lib/controller/accountscontroller.php

```diff
@@ -25,6 +25,8 @@
 
 namespace OCA\Mail\Controller;
 
+use Exception;
```

Comment:
since we are using this...we can change \Exception $ex to Exception $ex on line 236